frontend/packages/topology/src/components/index.ts

```diff
@@ -5,3 +5,5 @@ export { default as ConnectorArrow } from './ConnectorArrow';
 export { default as EdgeConnectorArrow } from './EdgeConnectorArrow';
 export { default as GraphComponent } from './GraphComponent';
 export { default as VisualizationSurface } from './VisualizationSurface';
+export * from './CreateConnectorWidget';
```

Comment:
Do not expose contents of `CreateConnectorWidget` as API. This is for internal use only to `withCreateConnector`. If needed, re-export any specific types from `withCreateConnector`.